README.md

```diff
@@ -86,6 +87,7 @@ The Helm Dashboard is being used and trusted by enterprises and communities all
 ## 👥 Community & Support
  
 Get updates and chat with project maintainers, contributors, and community members
+- [Documentation](https://docs.devtron.ai/getting-started/install/install-devtron)
```

Comment:
add a line space before pointers and change the hyperlink URL to `https://www.docs.devtron.ai/`. Also, make a full sentence. like -

- Feel free to check out the [Documentation](https://www.docs.devtron.ai/)

Reply:
I updated the hyperlink as suggested.